cmd/main.go

```diff
@@ -92,6 +91,14 @@ func main() {
 		glog.Fatal("env variable NODE_NAME must be set so that this provisioner can identify itself")
 	}
 
+	// Decides what type of provider to deploy, either block or fss
+	provisionerType := os.Getenv("PROVISIONER_TYPE")
```

Comment:
I thought we decided that we were going to run both provisioners by default and allow disabling specific provisioners behind a command line flag (see: [here](https://github.com/kubernetes/kubernetes/blob/3b4eb0a4a03a62e069d7bf7e00620a7dbff7ffdd/cmd/kube-controller-manager/app/options/options.go#L263)).

Reply:
I spoke to @owainlewis and he was more in favour of having separate provisioners

Reply:
Let's discuss offline then